mewarpx/mewarpx/diags_store/particle_diagnostic.py

```diff
@@ -112,38 +112,42 @@ def check_for_end_of_sim(self):
     def do_post_processing(self):
         if mwxrun.me == 0:
             data_dirs = glob.glob(os.path.join(
-                self.write_dir, self.name + '*'))
+                self.write_dir, self.name + '*'
+            ))
 
             if len(data_dirs) == 0:
                 raise RuntimeError(f'No diagnostic data '
                                    f'found in {self.write_dir}')
 
             for datafolder in data_dirs:
-                print('Reading', datafolder, '\n')
-                step = datafolder.split("_")[-1]
-                yt_data = yt.load(datafolder)
-                grid_data = yt_data.covering_grid(
-                    level=0,
-                    left_edge=yt_data.domain_left_edge,
-                    dims=yt_data.domain_dimensions
-                )
-                for species_name in self.plot_species:
-                    for param in self.plot_data_list:
-                        field = (species_name, param)
-                        if field not in yt_data.field_list:
-                            warnings.warn(f'{field} '
-                                           'not found in yt_data field list')
-                        else:
-                            raw_data = grid_data[(species_name, param)]
-                            np_data = raw_data.to_ndarray()
-                            fig, ax = plt.subplots(figsize=(14, 14))
-                            ax.hist(np_data, bins=500, histtype='stepfilled',
-                                    alpha=0.85)
-                            ax.set_xlabel(param)
-                            ax.set_ylabel("Counts")
-                            ax.set_title(species_name)
-                            out_path = os.path.join(
-                                self.write_dir,
-                                f'{species_name}_{param}_{step}.jpg',
-                            )
-                            plt.savefig(out_path)
+                if not "old" in datafolder:
+
```

Comment:
This is not a big thing but my preference is for cases like this where you have a lot of code inside the `if` condition, to rather skip the cases you don't want to do i.e. instead do
```
if "old" in datafolder:
    continue
```
and then unindent the lines below.